Adds imprint page (Iss #23)

f-lair opened the pull request:
The imprint page is added to the set of pages. It consists of the imprint section itself as well as a section containing the data privacy statement.

f-lair committed:
Initial commit

Added 'imprint' page based on 'group_creation' page.

f-lair committed:
Added imprint section

Added 'imprint' section (1/2) to 'imprint' page.

f-lair committed:
Added data privacy statement section

Added 'data privacy statement' section (2/2) to 'imprint' page.

DavidRisch committed:
Removed unnecessary parameters from page/imprint

DavidRisch commented on css/styles_general.css:
```diff
@@ -43,6 +45,39 @@ body {
+/* unvisited link */
+a:link {
+	text-decoration: none;
+	color: #ffffff;
```
A link should never look the same as regular text.
https://www.nngroup.com/articles/guidelines-for-visualizing-links/

f-lair committed:
Changed hyperlink appearance

In default case, Hyperlinks are colored turquoise. Interaction causes them to become underlined.
